lib/iris/tests/unit/cube_coord_common/test_get_valid_standard_name.py

```diff
@@ -19,10 +19,26 @@ class Test(tests.IrisTest):
     def setUp(self):
         self.emsg = "'{}' is not a valid standard_name"
 
+    def test_none_is_valid(self):
+        name = None
+        self.assertEqual(get_valid_standard_name(name), name)
+
+    def test_empty_is_not_valid(self):
+        name = ''
+        self.assertEqual(get_valid_standard_name(name), name)
+
+    def test_only_whitespace_is_not_valid(self):
```

Comment:
@jvegasbsc Same here, change test name to be `test_only_whitespace_passed_thru`...